tests/integration_runner.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Dogecoin Core developers
+"""
+Runs the integration tests
+"""
+
+import subprocess
+import sys
+
+from .integration.framework.test_runner import TestRunner
+
+class IntegrationRunner(TestRunner):
+    """Runs the integration tests"""
+
+    def __init__(self):
+        """Constructor"""
+        TestRunner.__init__(self)
+        self.found_failure = False
+        self.result_map = {}
+
+
+    def add_options(self, parser):
+        """Add test-specific --version option"""
+        parser.add_argument("--version", dest="version", required=True,
+            help="The version that is expected to be installed, eg: '1.14.5'")
+
+    def run_test(self):
+        """Run all specified tests and inherit any failures"""
+
+        tests = [
+            [ "version", [ "--version", self.options.version ] ]
```

Comment:
You should end lists which are updated regularly with `,`, it facilitates versioning.
```suggestion
            [ "version", [ "--version", self.options.version ] ],
```